src/Tribe/Utils/Array.php

```diff
@@ -1,9 +1,13 @@
 <?php
+
+if ( class_exists( 'Tribe__Utils__Array' ) ) {
```

Comment:
I seem to remember you have to wrap the whole class definition in order to use this kind of logic. Is that still true? Have you tested overriding the class with this way? Maybe I'm remembering the PHP 5.x days, was it fixed in 7.x+

Reply:
Adding a test for it.